notebooks/en/llm_judge_evaluating_ai_search_engines_with_judges_library.ipynb

```diff
@@ -0,0 +1,1680 @@
+{
```

Comment:
<p>It may be easier to consume this content in table-form</p><p> </p><p>| Judge | What | Why | Source | When to use |</p><p> </p><p>|---|---|---|---|---|</p><p> </p><p>| PollMultihopCorrectness |&nbsp;|&nbsp;|&nbsp;|&nbsp;|</p><p> </p><p>| PrometheusAbsoluteCoarseCorrectness |&nbsp;|&nbsp;|&nbsp;|&nbsp;|</p><p> </p><p>| MTBenchChatBotResponseQuality |&nbsp;|&nbsp;|&nbsp;|&nbsp;|</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>